include/seqan3/utility/char_operations/all.hpp

```diff
@@ -0,0 +1,21 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides Char predicates for tokenization.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/utility/char_operations/predicate.hpp>
+
+/*!\defgroup char_operations builtin_character operations
+ * \brief Provides various operations on character types.
+ * \ingroup core
```

Comment:
Should be in group utility right ?
Just stumbled upon this while searching for some inspiration 🧐
```suggestion
 * \ingroup utility
```

Reply:
yes thanks!